### PR TITLE
Huddle-up page

### LINKED ISSUE #42: HuddleUp Page

### PR DESCRIPTION
**Pull request by subho007**
Fixes #42

### CODE REVIEW

File: huddleup/assets/Gruntfile.js [hunk 1]
```diff
@@ -0,0 +1,99 @@
+'use strict';
```
Comment (subho007) [inline huddleup/assets/Gruntfile.js]: **[jshint]**
-  Use the function form of "use strict".

File: huddleup/assets/Gruntfile.js [hunk 2]
```diff
@@ -0,0 +1,99 @@
+'use strict';
+module.exports = function(grunt) {
```
Comment (subho007) [inline huddleup/assets/Gruntfile.js]: **[jshint]**
-  'module' is not defined.

File: huddleup/assets/js/html5shiv.js [hunk 3]
```diff
@@ -0,0 +1,8 @@
+/*
+ HTML5 Shiv v3.7.0 | @afarkas @jdalton @jon_neal @rem | MIT/GPL2 Licensed
+*/
+(function(l,f){function m(){var a=e.elements;return"string"==typeof a?a.split(" "):a}function i(a){var b=n[a[o]];b||(b={},h++,a[o]=h,n[h]=b);return b}function p(a,b,c){b||(b=f);if(g)return b.createElement(a);c||(c=i(b));b=c.cache[a]?c.cache[a].cloneNode():r.test(a)?(c.cache[a]=c.createElem(a)).cloneNode():c.createElem(a);return b.canHaveChildren&&!s.test(a)?c.frag.appendChild(b):b}function t(a,b){if(!b.cache)b.cache={},b.createElem=a.createElement,b.createFrag=a.createDocumentFragment,b.frag=b.createFrag();
```
Comment (subho007) [inline huddleup/assets/js/html5shiv.js]: **[jshint]**
-  Expected an assignment or function call and instead saw an expression.

File: huddleup/assets/js/html5shiv.js [hunk 4]
```diff
@@ -0,0 +1,8 @@
...
+*/
+(function(l,f){function m(){var a=e.elements;return"string"==typeof a?a.split(" "):a}function i(a){var b=n[a[o]];b||(b={},h++,a[o]=h,n[h]=b);return b}function p(a,b,c){b||(b=f);if(g)return b.createElement(a);c||(c=i(b));b=c.cache[a]?c.cache[a].cloneNode():r.test(a)?(c.cache[a]=c.createElem(a)).cloneNode():c.createElem(a);return b.canHaveChildren&&!s.test(a)?c.frag.appendChild(b):b}function t(a,b){if(!b.cache)b.cache={},b.createElem=a.createElement,b.createFrag=a.createDocumentFragment,b.frag=b.createFrag();
+a.createElement=function(c){return!e.shivMethods?b.createElem(c):p(c,a,b)};a.createDocumentFragment=Function("h,f","return function(){var n=f.cloneNode(),c=n.createElement;h.shivMethods&&("+m().join().replace(/[\w\-]+/g,function(a){b.createElem(a);b.frag.createElement(a);return'c("'+a+'")'})+");return n}")(e,b.frag)}function q(a){a||(a=f);var b=i(a);if(e.shivCSS&&!j&&!b.hasCSS){var c,d=a;c=d.createElement("p");d=d.getElementsByTagName("head")[0]||d.documentElement;c.innerHTML="x<style>article,aside,dialog,figcaption,figure,footer,header,hgroup,main,nav,section{display:block}mark{background:#FF0;color:#000}template{display:none}</style>";
```
Comment (subho007) [inline huddleup/assets/js/html5shiv.js]: **[jshint]**
-  eval can be harmful.
-  Expected an assignment or function call and instead saw an expression.

File: huddleup/assets/js/html5shiv.js [hunk 5]
```diff
@@ -0,0 +1,8 @@
...
+a.createElement=function(c){return!e.shivMethods?b.createElem(c):p(c,a,b)};a.createDocumentFragment=Function("h,f","return function(){var n=f.cloneNode(),c=n.createElement;h.shivMethods&&("+m().join().replace(/[\w\-]+/g,function(a){b.createElem(a);b.frag.createElement(a);return'c("'+a+'")'})+");return n}")(e,b.frag)}function q(a){a||(a=f);var b=i(a);if(e.shivCSS&&!j&&!b.hasCSS){var c,d=a;c=d.createElement("p");d=d.getElementsByTagName("head")[0]||d.documentElement;c.innerHTML="x<style>article,aside,dialog,figcaption,figure,footer,header,hgroup,main,nav,section{display:block}mark{background:#FF0;color:#000}template{display:none}</style>";
+c=d.insertBefore(c.lastChild,d.firstChild);b.hasCSS=!!c}g||t(a,b);return a}var k=l.html5||{},s=/^<|^(?:button|map|select|textarea|object|iframe|option|optgroup)$/i,r=/^(?:a|b|code|div|fieldset|h1|h2|h3|h4|h5|h6|i|label|li|ol|p|q|span|strong|style|table|tbody|td|th|tr|ul)$/i,j,o="_html5shiv",h=0,n={},g;(function(){try{var a=f.createElement("a");a.innerHTML="<xyz></xyz>";j="hidden"in a;var b;if(!(b=1==a.childNodes.length)){f.createElement("a");var c=f.createDocumentFragment();b="undefined"==typeof c.cloneNode||
```
Comment (subho007) [inline huddleup/assets/js/html5shiv.js]: **[jshint]**
-  Expected an assignment or function call and instead saw an expression.

File: huddleup/assets/js/html5shiv.js [hunk 6]
```diff
@@ -0,0 +1,8 @@
...
+c=d.insertBefore(c.lastChild,d.firstChild);b.hasCSS=!!c}g||t(a,b);return a}var k=l.html5||{},s=/^<|^(?:button|map|select|textarea|object|iframe|option|optgroup)$/i,r=/^(?:a|b|code|div|fieldset|h1|h2|h3|h4|h5|h6|i|label|li|ol|p|q|span|strong|style|table|tbody|td|th|tr|ul)$/i,j,o="_html5shiv",h=0,n={},g;(function(){try{var a=f.createElement("a");a.innerHTML="<xyz></xyz>";j="hidden"in a;var b;if(!(b=1==a.childNodes.length)){f.createElement("a");var c=f.createDocumentFragment();b="undefined"==typeof c.cloneNode||
+"undefined"==typeof c.createDocumentFragment||"undefined"==typeof c.createElement}g=b}catch(d){g=j=!0}})();var e={elements:k.elements||"abbr article aside audio bdi canvas data datalist details dialog figcaption figure footer header hgroup main mark meter nav output progress section summary template time video",version:"3.7.0",shivCSS:!1!==k.shivCSS,supportsUnknownElements:g,shivMethods:!1!==k.shivMethods,type:"default",shivDocument:q,createElement:p,createDocumentFragment:function(a,b){a||(a=f);
```
Comment (subho007) [inline huddleup/assets/js/html5shiv.js]: **[jshint]**
-  Confusing use of '!'.
-  Expected an assignment or function call and instead saw an expression.

File: huddleup/assets/js/html5shiv.js [hunk 7]
```diff
@@ -0,0 +1,8 @@
...
+c=d.insertBefore(c.lastChild,d.firstChild);b.hasCSS=!!c}g||t(a,b);return a}var k=l.html5||{},s=/^<|^(?:button|map|select|textarea|object|iframe|option|optgroup)$/i,r=/^(?:a|b|code|div|fieldset|h1|h2|h3|h4|h5|h6|i|label|li|ol|p|q|span|strong|style|table|tbody|td|th|tr|ul)$/i,j,o="_html5shiv",h=0,n={},g;(function(){try{var a=f.createElement("a");a.innerHTML="<xyz></xyz>";j="hidden"in a;var b;if(!(b=1==a.childNodes.length)){f.createElement("a");var c=f.createDocumentFragment();b="undefined"==typeof c.cloneNode||
+"undefined"==typeof c.createDocumentFragment||"undefined"==typeof c.createElement}g=b}catch(d){g=j=!0}})();var e={elements:k.elements||"abbr article aside audio bdi canvas data datalist details dialog figcaption figure footer header hgroup main mark meter nav output progress section summary template time video",version:"3.7.0",shivCSS:!1!==k.shivCSS,supportsUnknownElements:g,shivMethods:!1!==k.shivMethods,type:"default",shivDocument:q,createElement:p,createDocumentFragment:function(a,b){a||(a=f);
+if(g)return a.createDocumentFragment();for(var b=b||i(a),c=b.frag.cloneNode(),d=0,e=m(),h=e.length;d<h;d++)c.createElement(e[d]);return c}};l.html5=e;q(f)})(this,document);
```
Comment (subho007) [inline huddleup/assets/js/html5shiv.js]: **[jshint]**
-  'b' is already defined.

File: huddleup/assets/js/jquery.countdown.js [hunk 8]
```diff
@@ -0,0 +1,857 @@
...
+			inst = inst || elem.data(this.name);
+		if (!inst) {
+			return;
+		}
+			elem.html(this._generateHTML(inst)).toggleClass(this._rtlClass, inst.options.isRTL);
+		if ($.isFunction(inst.options.onTick)) {
+			var periods = inst._hold != 'lap' ? inst._periods :
+				this._calculatePeriods(inst, inst._show, inst.options.significant, new Date());
+			if (inst.options.tickInterval == 1 ||
+					this.periodsToSeconds(periods) % inst.options.tickInterval == 0) {
```
Comment (subho007) [inline huddleup/assets/js/jquery.countdown.js]: **[jshint]**
-  Use '===' to compare with '0'.

File: huddleup/assets/js/jquery.countdown.js [hunk 9]
```diff
@@ -0,0 +1,857 @@
...
+	_resetExtraLabels: function(base, options) {
+		var changingLabels = false;
+		for (var n in options) {
+			if (n != 'whichLabels' && n.match(/[Ll]abels/)) {
+				changingLabels = true;
+				break;
+			}
+		}
+		if (changingLabels) {
+			for (var n in base) { // Remove custom numbered labels
```
Comment (subho007) [inline huddleup/assets/js/jquery.countdown.js]: **[jshint]**
-  'n' is already defined.

File: huddleup/assets/js/jquery.countdown.js [hunk 10]
```diff
@@ -0,0 +1,857 @@
...
+		var sigCount = inst.options.significant;
+		var show = $.extend({}, inst._show);
+		for (var period = Y; period <= S; period++) {
+			shownNonZero |= (inst._show[period] == '?' && inst._periods[period] > 0);
+			show[period] = (inst._show[period] == '?' && !shownNonZero ? null : inst._show[period]);
+			showCount += (show[period] ? 1 : 0);
+			sigCount -= (inst._periods[period] > 0 ? 1 : 0);
+		}
+		var showSignificant = [false, false, false, false, false, false, false];
+		for (var period = S; period >= Y; period--) { // Determine significant periods
```
Comment (subho007) [inline huddleup/assets/js/jquery.countdown.js]: **[jshint]**
-  'period' is already defined.

File: huddleup/assets/js/jquery.countdown.js [hunk 11]
```diff
@@ -0,0 +1,857 @@
...
+						diff = 0;
+						lastShown = period;
+						max = 1;
+					}
+				}
+				max *= multiplier[period];
+			}
+		}
+		if (significant) { // Zero out insignificant periods
+			for (var period = Y; period <= S; period++) {
```
Comment (subho007) [inline huddleup/assets/js/jquery.countdown.js]: **[jshint]**
-  'period' is already defined.

File: huddleup/assets/js/jquery.flexslider-min.js [hunk 12]
```diff
@@ -0,0 +1,5 @@
...
...deshow:!0,slideshowSpeed:7e3,animationSpeed:600,initDelay:0,randomize:!1,thumbCaptions:!1,pauseOnAction:!0,pauseOnHover:!1,pauseInvisible:!0,useCSS:!0,touch:!0,video:!1,controlNav:!0,directionNav:!0,prevText:"Previous",nextText:"Next",keyboard:!0,multipleKeyboard:!1,mousewheel:!1,pausePlay:!1,pauseText:"Pause",playText:"Play",controlsContainer:"",manualControls:"",sync:"",asNavFor:"",itemWidth:0,itemMargin:0,minItems:1,maxItems:0,move:0,allowOneSlide:!0,start:function(){},before:function(){},after:function(){},end:function(){},added:function(){},removed:function(){}};e.fn.flexslider=function(t){t===undefined&&(t={});if(typeof t=="object")return this.each(function(){var n=e(this),r=t.selector?t.selector:".slides > li",i=n.find(r);if(i.length===1&&t.allowOneSlide===!0||i.length===0){i.fadeIn(400);t.start&&t.start(n)}else n.data("flexslider")===undefined&&new e.flexslider(this,t)});var n=e(this).data("flexslider");switch(t){case"play":n.play();break;case"pause":n.pause();break;case"stop":n.stop();break;case"next":n.flexAnimate(n.getTarget("next"),!0);break;case"prev":case"previous":n.flexAnimate(n.getTarget("prev"),!0);break;default:typeof t=="number"&&n.flexAnimate(t,!0)}}})(jQuery);
```
Comment (subho007) [inline huddleup/assets/js/jquery.flexslider-min.js]: **[jshint]**
-  A leading decimal point can be confused with a dot
-  Expected an assignment or function call and instead saw an expression.
-  Missing '()' invoking a constructor.
-  Confusing use of '!'.
-  Use '!==' to compare with ''.
-  Use '!==' to compare with 'undefined'.
-  Function declarations should not be placed in blocks. Use a function expression or move the statement to the top of the outer function.
-  Too many errors. (100% scanned).

File: huddleup/assets/js/jquery.plugin.js [hunk 13]
```diff
@@ -0,0 +1,344 @@
...
+		initializing = true;
+		var prototype = new this();
+		initializing = false;
+
+		// Copy the properties over onto the new prototype
+		for (var name in prop) {
+			// Check if we're overwriting an existing function
+			prototype[name] = typeof prop[name] == 'function' &&
+				typeof base[name] == 'function' ?
+				(function(name, fn){
```
Comment (subho007) [inline huddleup/assets/js/jquery.plugin.js]: **[jshint]**
-  Don't make functions within a loop.

File: huddleup/assets/js/jquery.plugin.js [hunk 14]
```diff
@@ -0,0 +1,344 @@
...
+				data = data.replace(/'/g, '"');
+				data = data.replace(/([a-zA-Z0-9]+):/g, function(match, group, i) { 
+					var count = data.substring(0, i).match(/"/g); // Handle embedded ':'
+					return (!count || count.length % 2 === 0 ? '"' + group + '":' : group + ':');
+				});
+				data = $.parseJSON('{' + data + '}');
+				for (var name in data) { // Convert dates
+					var value = data[name];
+					if (typeof value === 'string' && value.match(/^new Date\((.*)\)$/)) {
+						data[name] = eval(value);
```
Comment (subho007) [inline huddleup/assets/js/jquery.plugin.js]: **[jshint]**
-  eval can be harmful.

File: huddleup/assets/js/jquery.plugin.js [hunk 15]
```diff
@@ -0,0 +1,344 @@
...
+			elem = $(elem);
+			var inst = elem.data(this.name);
+			if  (!name || (typeof name === 'string' && value == null)) {
+				var options = (inst || {}).options;
+				return (options && name ? options[name] : options);
+			}
+			if (!elem.hasClass(this._getMarker())) {
+				return;
+			}
+			var options = name || {};
```
Comment (subho007) [inline huddleup/assets/js/jquery.plugin.js]: **[jshint]**
-  'options' is already defined.

File: huddleup/assets/js/ventcamp.js [hunk 16]
```diff
@@ -0,0 +1,1220 @@
...
+                        $responseBlock = $form.find('.response'),
+                        email = $emailField.val(),
+                        first_name = '',
+                        last_name = '';
+
+                    if ($(this).find('[name=NewsletterName]') .length > 0){
+                        $fullnameField = $form.find('[name=NewsletterName]');
+                        fullname = $fullnameField.val().split(' '), // Gwt Full name and split it by space
+                        first_name = fullname[0], // First part is the first name
+                        last_name = fullname[1]; // Second part is Last name
```
Comment (subho007) [inline huddleup/assets/js/ventcamp.js]: **[jshint]**
-  Expected an assignment or function call and instead saw an expression.

File: huddleup/assets/js/ventcamp.js [hunk 17]
```diff
@@ -0,0 +1,1220 @@
...
+                        if ( !last_name ) last_name = ''; //By chance they can input only their first name, then we should null the last name since there is none
+                    } else {
+                        first_name = '',
+                        last_name = '';
```
Comment (subho007) [inline huddleup/assets/js/ventcamp.js]: **[jshint]**
-  Expected an assignment or function call and instead saw an expression.

File: huddleup/assets/js/ventcamp.js [hunk 18]
```diff
@@ -0,0 +1,1220 @@
...
+                            var balloon = balloons[i];
+
+                            if ( typeof balloon == 'string' ) {
+                                service.textSearch({
+                                    location: markerLatLng,
+                                    radius: 5000,
+                                    query: balloon
+                                }, function(results, status) {
+                                    if (status == google.maps.places.PlacesServiceStatus.OK) {
+                                        for (var i = 0, place; place = results[i]; i++) {
```
Comment (subho007) [inline huddleup/assets/js/ventcamp.js]: **[jshint]**
-  Expected a conditional expression and instead saw an assignment.

File: huddleup/assets/js/ventcamp.js [hunk 19]
```diff
@@ -0,0 +1,1220 @@
...
+                                            service.getDetails({placeId: place.place_id}, createMarkers);
+                                        };
```
Comment (subho007) [inline huddleup/assets/js/ventcamp.js]: **[jshint]**
-  Unnecessary semicolon.

File: huddleup/assets/js/ventcamp.js [hunk 20]
```diff
@@ -0,0 +1,1220 @@
...
+                    if ( balloons ) {
+                        for (var i = 0; i < balloons.length; i++) {
+                            var balloon = balloons[i];
+
+                            if ( typeof balloon == 'string' ) {
+                                service.textSearch({
+                                    location: markerLatLng,
+                                    radius: 5000,
+                                    query: balloon
+                                }, function(results, status) {
```
Comment (subho007) [inline huddleup/assets/js/ventcamp.js]: **[jshint]**
-  Don't make functions within a loop.

File: huddleup/assets/js/ventcamp.js [hunk 21]
```diff
@@ -0,0 +1,1220 @@
...
+                                    if (status == google.maps.places.PlacesServiceStatus.OK) {
+                                        for (var i = 0, place; place = results[i]; i++) {
+                                            service.getDetails({placeId: place.place_id}, createMarkers);
+                                        };
+                                    }
+                                });
+                            }
+                        };
```
Comment (subho007) [inline huddleup/assets/js/ventcamp.js]: **[jshint]**
-  Unnecessary semicolon.

File: huddleup/assets/js/ventcamp.js [hunk 22]
```diff
@@ -0,0 +1,1220 @@
...
+                        var innerContent;
+
+                        innerContent = '<div class="balloon"><strong class="name">' + place.name + '</strong>';
+
+                        if ( place.rating ) {
+                            innerContent += '<ul class="list-inline rating">';
+
+                            for (var i = 0; i < Math.floor(place.rating); i++) {
+                                innerContent += '<li><span class="fa fa-star"></span></li>'
+                            };
```
Comment (subho007) [inline huddleup/assets/js/ventcamp.js]: **[jshint]**
-  Unnecessary semicolon.

File: huddleup/assets/js/ventcamp.js [hunk 23]
```diff
@@ -0,0 +1,1220 @@
...
+                            content: innerContent,
+                            disableAutoPan: true
+                        });
+
+                        infowindow.open(map);
+                    }
+                }
+
+                function centerMap(map, offsetX, offsetY, relative) {
+                    var offsetX = (typeof offsetX == 'number' ? offsetX : 0),
```
Comment (subho007) [inline huddleup/assets/js/ventcamp.js]: **[jshint]**
-  'offsetX' is already defined.

File: huddleup/assets/js/ventcamp.js [hunk 24]
```diff
@@ -0,0 +1,1220 @@
...
+                        offsetY = (typeof offsetY == 'number' ? offsetY : 0),
```
Comment (subho007) [inline huddleup/assets/js/ventcamp.js]: **[jshint]**
-  'offsetY' is already defined.

File: huddleup/assets/js/ventcamp.js [hunk 25]
```diff
@@ -0,0 +1,1220 @@
...
+                    var $this = $(this),
+                        $li = $this.parent(),
+                        imgSrc = $this.attr('src');
+
+                    $li.css('background-image', 'url("' + imgSrc + '")').data('imgSrc', imgSrc);
+                    $this.remove();
+                });
+
+                if ( $imgSLider.length && $descrSlider.length ) {
+                    function countImgMinHeight ($slide) {
```
Comment (subho007) [inline huddleup/assets/js/ventcamp.js]: **[jshint]**
-  Function declarations should not be placed in blocks. Use a function expression or move the statement to the top of the outer function.

File: js/html5shiv.js [hunk 26]
```diff
@@ -0,0 +1,8 @@
+/*
+ HTML5 Shiv v3.7.0 | @afarkas @jdalton @jon_neal @rem | MIT/GPL2 Licensed
+*/
+(function(l,f){function m(){var a=e.elements;return"string"==typeof a?a.split(" "):a}function i(a){var b=n[a[o]];b||(b={},h++,a[o]=h,n[h]=b);return b}function p(a,b,c){b||(b=f);if(g)return b.createElement(a);c||(c=i(b));b=c.cache[a]?c.cache[a].cloneNode():r.test(a)?(c.cache[a]=c.createElem(a)).cloneNode():c.createElem(a);return b.canHaveChildren&&!s.test(a)?c.frag.appendChild(b):b}function t(a,b){if(!b.cache)b.cache={},b.createElem=a.createElement,b.createFrag=a.createDocumentFragment,b.frag=b.createFrag();
```
Comment (subho007) [inline js/html5shiv.js:4]: **[jshint]**
-  Expected an assignment or function call and instead saw an expression.

File: js/html5shiv.js [hunk 27]
```diff
@@ -0,0 +1,8 @@
...
+*/
+(function(l,f){function m(){var a=e.elements;return"string"==typeof a?a.split(" "):a}function i(a){var b=n[a[o]];b||(b={},h++,a[o]=h,n[h]=b);return b}function p(a,b,c){b||(b=f);if(g)return b.createElement(a);c||(c=i(b));b=c.cache[a]?c.cache[a].cloneNode():r.test(a)?(c.cache[a]=c.createElem(a)).cloneNode():c.createElem(a);return b.canHaveChildren&&!s.test(a)?c.frag.appendChild(b):b}function t(a,b){if(!b.cache)b.cache={},b.createElem=a.createElement,b.createFrag=a.createDocumentFragment,b.frag=b.createFrag();
+a.createElement=function(c){return!e.shivMethods?b.createElem(c):p(c,a,b)};a.createDocumentFragment=Function("h,f","return function(){var n=f.cloneNode(),c=n.createElement;h.shivMethods&&("+m().join().replace(/[\w\-]+/g,function(a){b.createElem(a);b.frag.createElement(a);return'c("'+a+'")'})+");return n}")(e,b.frag)}function q(a){a||(a=f);var b=i(a);if(e.shivCSS&&!j&&!b.hasCSS){var c,d=a;c=d.createElement("p");d=d.getElementsByTagName("head")[0]||d.documentElement;c.innerHTML="x<style>article,aside,dialog,figcaption,figure,footer,header,hgroup,main,nav,section{display:block}mark{background:#FF0;color:#000}template{display:none}</style>";
```
Comment (subho007) [inline js/html5shiv.js:5]: **[jshint]**
-  eval can be harmful.
-  Expected an assignment or function call and instead saw an expression.

File: js/html5shiv.js [hunk 28]
```diff
@@ -0,0 +1,8 @@
...
+a.createElement=function(c){return!e.shivMethods?b.createElem(c):p(c,a,b)};a.createDocumentFragment=Function("h,f","return function(){var n=f.cloneNode(),c=n.createElement;h.shivMethods&&("+m().join().replace(/[\w\-]+/g,function(a){b.createElem(a);b.frag.createElement(a);return'c("'+a+'")'})+");return n}")(e,b.frag)}function q(a){a||(a=f);var b=i(a);if(e.shivCSS&&!j&&!b.hasCSS){var c,d=a;c=d.createElement("p");d=d.getElementsByTagName("head")[0]||d.documentElement;c.innerHTML="x<style>article,aside,dialog,figcaption,figure,footer,header,hgroup,main,nav,section{display:block}mark{background:#FF0;color:#000}template{display:none}</style>";
+c=d.insertBefore(c.lastChild,d.firstChild);b.hasCSS=!!c}g||t(a,b);return a}var k=l.html5||{},s=/^<|^(?:button|map|select|textarea|object|iframe|option|optgroup)$/i,r=/^(?:a|b|code|div|fieldset|h1|h2|h3|h4|h5|h6|i|label|li|ol|p|q|span|strong|style|table|tbody|td|th|tr|ul)$/i,j,o="_html5shiv",h=0,n={},g;(function(){try{var a=f.createElement("a");a.innerHTML="<xyz></xyz>";j="hidden"in a;var b;if(!(b=1==a.childNodes.length)){f.createElement("a");var c=f.createDocumentFragment();b="undefined"==typeof c.cloneNode||
```
Comment (subho007) [inline js/html5shiv.js:6]: **[jshint]**
-  Expected an assignment or function call and instead saw an expression.

File: js/html5shiv.js [hunk 29]
```diff
@@ -0,0 +1,8 @@
...
+c=d.insertBefore(c.lastChild,d.firstChild);b.hasCSS=!!c}g||t(a,b);return a}var k=l.html5||{},s=/^<|^(?:button|map|select|textarea|object|iframe|option|optgroup)$/i,r=/^(?:a|b|code|div|fieldset|h1|h2|h3|h4|h5|h6|i|label|li|ol|p|q|span|strong|style|table|tbody|td|th|tr|ul)$/i,j,o="_html5shiv",h=0,n={},g;(function(){try{var a=f.createElement("a");a.innerHTML="<xyz></xyz>";j="hidden"in a;var b;if(!(b=1==a.childNodes.length)){f.createElement("a");var c=f.createDocumentFragment();b="undefined"==typeof c.cloneNode||
+"undefined"==typeof c.createDocumentFragment||"undefined"==typeof c.createElement}g=b}catch(d){g=j=!0}})();var e={elements:k.elements||"abbr article aside audio bdi canvas data datalist details dialog figcaption figure footer header hgroup main mark meter nav output progress section summary template time video",version:"3.7.0",shivCSS:!1!==k.shivCSS,supportsUnknownElements:g,shivMethods:!1!==k.shivMethods,type:"default",shivDocument:q,createElement:p,createDocumentFragment:function(a,b){a||(a=f);
```
Comment (subho007) [inline js/html5shiv.js:7]: **[jshint]**
-  Confusing use of '!'.
-  Expected an assignment or function call and instead saw an expression.

File: js/html5shiv.js [hunk 30]
```diff
@@ -0,0 +1,8 @@
...
+c=d.insertBefore(c.lastChild,d.firstChild);b.hasCSS=!!c}g||t(a,b);return a}var k=l.html5||{},s=/^<|^(?:button|map|select|textarea|object|iframe|option|optgroup)$/i,r=/^(?:a|b|code|div|fieldset|h1|h2|h3|h4|h5|h6|i|label|li|ol|p|q|span|strong|style|table|tbody|td|th|tr|ul)$/i,j,o="_html5shiv",h=0,n={},g;(function(){try{var a=f.createElement("a");a.innerHTML="<xyz></xyz>";j="hidden"in a;var b;if(!(b=1==a.childNodes.length)){f.createElement("a");var c=f.createDocumentFragment();b="undefined"==typeof c.cloneNode||
+"undefined"==typeof c.createDocumentFragment||"undefined"==typeof c.createElement}g=b}catch(d){g=j=!0}})();var e={elements:k.elements||"abbr article aside audio bdi canvas data datalist details dialog figcaption figure footer header hgroup main mark meter nav output progress section summary template time video",version:"3.7.0",shivCSS:!1!==k.shivCSS,supportsUnknownElements:g,shivMethods:!1!==k.shivMethods,type:"default",shivDocument:q,createElement:p,createDocumentFragment:function(a,b){a||(a=f);
+if(g)return a.createDocumentFragment();for(var b=b||i(a),c=b.frag.cloneNode(),d=0,e=m(),h=e.length;d<h;d++)c.createElement(e[d]);return c}};l.html5=e;q(f)})(this,document);
```
Comment (subho007) [inline js/html5shiv.js:8]: **[jshint]**
-  'b' is already defined.

File: js/jquery.plugin.js [hunk 31]
```diff
@@ -0,0 +1,344 @@
...
+		initializing = true;
+		var prototype = new this();
+		initializing = false;
+
+		// Copy the properties over onto the new prototype
+		for (var name in prop) {
+			// Check if we're overwriting an existing function
+			prototype[name] = typeof prop[name] == 'function' &&
+				typeof base[name] == 'function' ?
+				(function(name, fn){
```
Comment (subho007) [inline js/jquery.plugin.js:30]: **[jshint]**
-  Don't make functions within a loop.

File: js/jquery.plugin.js [hunk 32]
```diff
@@ -0,0 +1,344 @@
...
+				data = data.replace(/'/g, '"');
+				data = data.replace(/([a-zA-Z0-9]+):/g, function(match, group, i) { 
+					var count = data.substring(0, i).match(/"/g); // Handle embedded ':'
+					return (!count || count.length % 2 === 0 ? '"' + group + '":' : group + ':');
+				});
+				data = $.parseJSON('{' + data + '}');
+				for (var name in data) { // Convert dates
+					var value = data[name];
+					if (typeof value === 'string' && value.match(/^new Date\((.*)\)$/)) {
+						data[name] = eval(value);
```
Comment (subho007) [inline js/jquery.plugin.js:224]: **[jshint]**
-  eval can be harmful.

File: js/jquery.plugin.js [hunk 33]
```diff
@@ -0,0 +1,344 @@
...
+			elem = $(elem);
+			var inst = elem.data(this.name);
+			if  (!name || (typeof name === 'string' && value == null)) {
+				var options = (inst || {}).options;
+				return (options && name ? options[name] : options);
+			}
+			if (!elem.hasClass(this._getMarker())) {
+				return;
+			}
+			var options = name || {};
```
Comment (subho007) [inline js/jquery.plugin.js:260]: **[jshint]**
-  'options' is already defined.

File: js/ventcamp.js [hunk 34]
```diff
@@ -0,0 +1,1220 @@
...
+                        $responseBlock = $form.find('.response'),
+                        email = $emailField.val(),
+                        first_name = '',
+                        last_name = '';
+
+                    if ($(this).find('[name=NewsletterName]') .length > 0){
+                        $fullnameField = $form.find('[name=NewsletterName]');
+                        fullname = $fullnameField.val().split(' '), // Gwt Full name and split it by space
+                        first_name = fullname[0], // First part is the first name
+                        last_name = fullname[1]; // Second part is Last name
```
Comment (subho007) [inline js/ventcamp.js:538]: **[jshint]**
-  Expected an assignment or function call and instead saw an expression.

File: js/ventcamp.js [hunk 35]
```diff
@@ -0,0 +1,1220 @@
...
+                        if ( !last_name ) last_name = ''; //By chance they can input only their first name, then we should null the last name since there is none
+                    } else {
+                        first_name = '',
+                        last_name = '';
```
Comment (subho007) [inline js/ventcamp.js:542]: **[jshint]**
-  Expected an assignment or function call and instead saw an expression.

File: js/ventcamp.js [hunk 36]
```diff
@@ -0,0 +1,1220 @@
...
+                            var balloon = balloons[i];
+
+                            if ( typeof balloon == 'string' ) {
+                                service.textSearch({
+                                    location: markerLatLng,
+                                    radius: 5000,
+                                    query: balloon
+                                }, function(results, status) {
+                                    if (status == google.maps.places.PlacesServiceStatus.OK) {
+                                        for (var i = 0, place; place = results[i]; i++) {
```
Comment (subho007) [inline js/ventcamp.js:627]: **[jshint]**
-  Expected a conditional expression and instead saw an assignment.

File: js/ventcamp.js [hunk 37]
```diff
@@ -0,0 +1,1220 @@
...
+                                            service.getDetails({placeId: place.place_id}, createMarkers);
+                                        };
```
Comment (subho007) [inline js/ventcamp.js:629]: **[jshint]**
-  Unnecessary semicolon.

File: js/ventcamp.js [hunk 38]
```diff
@@ -0,0 +1,1220 @@
...
+                    if ( balloons ) {
+                        for (var i = 0; i < balloons.length; i++) {
+                            var balloon = balloons[i];
+
+                            if ( typeof balloon == 'string' ) {
+                                service.textSearch({
+                                    location: markerLatLng,
+                                    radius: 5000,
+                                    query: balloon
+                                }, function(results, status) {
```
Comment (subho007) [inline js/ventcamp.js:625]: **[jshint]**
-  Don't make functions within a loop.

File: js/ventcamp.js [hunk 39]
```diff
@@ -0,0 +1,1220 @@
...
+                                    if (status == google.maps.places.PlacesServiceStatus.OK) {
+                                        for (var i = 0, place; place = results[i]; i++) {
+                                            service.getDetails({placeId: place.place_id}, createMarkers);
+                                        };
+                                    }
+                                });
+                            }
+                        };
```
Comment (subho007) [inline js/ventcamp.js:633]: **[jshint]**
-  Unnecessary semicolon.

File: js/ventcamp.js [hunk 40]
```diff
@@ -0,0 +1,1220 @@
...
+                        var innerContent;
+
+                        innerContent = '<div class="balloon"><strong class="name">' + place.name + '</strong>';
+
+                        if ( place.rating ) {
+                            innerContent += '<ul class="list-inline rating">';
+
+                            for (var i = 0; i < Math.floor(place.rating); i++) {
+                                innerContent += '<li><span class="fa fa-star"></span></li>'
+                            };
```
Comment (subho007) [inline js/ventcamp.js:648]: **[jshint]**
-  Unnecessary semicolon.

File: js/ventcamp.js [hunk 41]
```diff
@@ -0,0 +1,1220 @@
...
+                            content: innerContent,
+                            disableAutoPan: true
+                        });
+
+                        infowindow.open(map);
+                    }
+                }
+
+                function centerMap(map, offsetX, offsetY, relative) {
+                    var offsetX = (typeof offsetX == 'number' ? offsetX : 0),
```
Comment (subho007) [inline js/ventcamp.js:675]: **[jshint]**
-  'offsetX' is already defined.

File: js/ventcamp.js [hunk 42]
```diff
@@ -0,0 +1,1220 @@
...
+                        offsetY = (typeof offsetY == 'number' ? offsetY : 0),
```
Comment (subho007) [inline js/ventcamp.js:676]: **[jshint]**
-  'offsetY' is already defined.

File: js/ventcamp.js [hunk 43]
```diff
@@ -0,0 +1,1220 @@
...
+                    var $this = $(this),
+                        $li = $this.parent(),
+                        imgSrc = $this.attr('src');
+
+                    $li.css('background-image', 'url("' + imgSrc + '")').data('imgSrc', imgSrc);
+                    $this.remove();
+                });
+
+                if ( $imgSLider.length && $descrSlider.length ) {
+                    function countImgMinHeight ($slide) {
```
Comment (subho007) [inline js/ventcamp.js:830]: **[jshint]**
-  Function declarations should not be placed in blocks. Use a function expression or move the statement to the top of the outer function.

**Comment by subho007** [comment]
@yashwin no updates on this branch. Speaker description, event descriptions nothing updated. Can you check with the marketing team for this ?

**Comment by yashwin** [comment]
@subho007 : No updates on this yet. I will make the changes once I get it from the marketing team.

**Pull request closed without merging**